### ISSUE TITLE
[Master feature] Default loading state improvement for all components

### ISSUE THREAD

**Issue by ericlindley-g** [ampproject/amphtml#14064]
Feature description:

AMP has a default loader for most content that depends on an external server call for content. Currently it's an animated ellipsis for most cases, and a more polished design for ads. This feature is to create a more polished design across all components that need it, and to possibly differentiate designs based on the context.

Related issues:
- [ ] Figure out which components use the default loading state. 

Related PRs:
/cc @aghassemi can you add relevant PRs when available? (either listing or linking by mentioning this feature in those PRs)

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @cathyxz Please add an appropriate category.

**Comment by ericlindley-g** [comment]
Related: https://github.com/ampproject/amphtml/issues/16313

**Comment by aghassemi** [comment]
#20237

**Comment by nainar** [comment]
Closing as this has launched. Please fill issues regarding this feature as a new issue.
